service/pricing/cid_gravity.go

```diff
@@ -141,9 +142,9 @@ func (cg *clientRules) maybeReloadRules(url string, timeout time.Duration, cache
 	if time.Since(lastUpdated) < cachePeriod {
 		return true
 	}
-	if t := cg.apiRateLimitReset.Load(); t != nil {
+	if t := cg.apiThrottlingReset.Load(); t != nil {
 		reset := t.(time.Time)
-		if time.Now().Before(reset) {
+		if !reset.IsZero() && time.Now().Before(reset) {
```

Comment:
I think there was a bug here since the "reset" atomic was never cleared out.
Possibly related with the rate limiting weirdness that somebody told some days ago?

cc @merlinran if you're around to comment.

Reply:
It doesn't matter if the atomic is cleared or not as long as `time.Now()` is after it, unless I missed something? The rate limiting problem some days ago is a CID gravity server side issue I think.

Reply:
Ah yeah, an old value should be fine.
Yep, the rate limiting headers will be removed and the rate limiting of the API will be done by a reverse proxy that simply don't set the headers.

Reply:
I'll remove the zeroing!